src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier color = new ColorSupplier();
+        return new Ball(color.getRandomColor(), new Random().nextInt(100));

```

Comment:
100 is magic number. Move him to the constant.

![image](https://user-images.githubusercontent.com/12864128/217532115-d340979e-7808-43a0-be24-4a3bdfe904bc.png)